src/service/mod.rs

```diff
@@ -0,0 +1,152 @@
+use rocket::async_trait;
+
+pub mod name;
+pub mod s3;
+
+#[async_trait]
+pub trait KeplerService {
```

Comment:
I think this might be mixing too much the business logic and runtime organisation. In the context of a monolith I would be in favour of letting Rocket manage those objects, or rather rely on the caching/retrieval abstraction we have for orbits, and rely on core traits like `Drop`.

Reply:
I've removed this trait in favour of a more basic impl.